static/src/javascripts/projects/common/modules/commercial/acquisitions-copy.js

```diff
@@ -32,96 +39,81 @@ const ctaLinkSentenceWorldCupLiveblogPlayful = (
 ): string =>
     `<span class="contributions__highlight"> For as little as ${currencySymbol}1, you can support the Guardian – and it only takes a minute.</span> Otherwise we’ll send Sergio Ramos round to sort you out. <a href="${supportUrl}" target="_blank" class="u-underline">Make a contribution</a>`;
 
-const thankyouP2FirstSentence =
-    'Thank you to the many people who have already supported us financially &ndash; your contribution is what makes stories like you’ve just read possible. We increasingly need our readers to fund our work so that we can continue holding power to account and producing fearless journalism.</br></br>';
+export const getCopyFromGoogleDoc = (
+    sheetName: string
+): Promise<AcquisitionsEpicTemplateCopy> => {
+    const url =
+        'https://interactive.guim.co.uk/docsdata/1fy0JolB1bf1IEFLHGHfUYWx-niad7vR9K954OpTOvjE.json';
+
+    return fetchJSON(url, {
+        mode: 'cors',
+    }).then(res => {
+        const rows = res && res.sheets && res.sheets[sheetName];
+        const firstRow = rows && rows[0];
+
+        if (
+            !(
+                firstRow &&
+                firstRow.heading &&
+                firstRow.paragraphs &&
+                firstRow.highlightedText
+            )
+        ) {
+            reportError(
+                new Error('Could not fetch epic copy from Google Doc'),
```

Comment:
Probably not for this PR, but this is currently throwing errors in the console when called from the `acquisitions-epic-from-google-doc-four-variants` and `acquisitions-epic-from-google-doc-five-variants` tests - we should fix this at some point

Reply:
yep, though usually we'd only have one test switched on and this would correspond to the variants in the sheet

Reply:
Yes, as is the case now, but despite the tests not being on it still calls the `getCopyFromGoogleDoc` function for each variant of all the tests

Reply:
Do we have any alarms set up on this error logging? It could really mess up a test if we didn't notice that this was failing (which seems like it could happen quite easily due to the nature of the data entry into the sheets).

Reply:
> Yes, as is the case now, but despite the tests not being on it still calls the `getCopyFromGoogleDoc` function for each variant of all the tests

You're absolutely right! I'd forgotten that request isn't made inside the test function, it's actually made as the test is being defined. That probably needs a rethink

Reply:
> Do we have any alarms set up on this error logging? It could really mess up a test if we didn't notice that this was failing (which seems like it could happen quite easily due to the nature of the data entry into the sheets).

No, but that's a good idea. They'll go into Sentry right now - maybe we can set something up in there

Reply:
The spurious errors should be resolved now so we're good to try and create an alarm. The tricky thing will be knowing what the right threshold is (there will almost certainly be some browsers or situations where the request fails for some reason but we won't want to do anything about it). As discussed, @michaelwmcnamara and the dotcom team might have some thoughts.

https://sentry.io/the-guardian/client-side-prod/?query=epic

There are a lot there! But that could be due to the issue you identified, now fixed